orc8r/cloud/go/obsidian/swagger/handlers/handlers.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package handlers
+
+import (
+	"net/http"
+
+	"magma/orc8r/cloud/go/obsidian"
+	"magma/orc8r/cloud/go/obsidian/swagger"
+
+	"github.com/labstack/echo"
+)
+
+// GenerateCombinedSpecHandler is a routing handler which creates and serves the
+// combined Swagger Spec.
+func GenerateCombinedSpecHandler(c echo.Context) error {
+	yamlCommon, err := swagger.GetCommonSpec()
```

Comment:
Hoist the reading of GetCommonSpec into the server.go so that we only ever have to do it once

Reply:
Hmm I was thinking the same thing but I couldn't see a easy way to store the common spec? Would we just have it as a global var in server.go?